tests/test_visitors/test_ast/test_redundancy/test_redundant_ternary.py

```diff
@@ -78,3 +86,23 @@ def test_wrong_ternary(
     visitor.run()
 
     assert_errors(visitor, [RedundantTernaryViolation])
+
+
+@pytest.mark.parametrize('code', [
+    non_compare,
+    two_operation_compare,
+])
+def safety_check_ternary(
+    code,
+    assert_errors,
+    parse_ast_tree,
+    default_options,
+    mode,
```

Comment:
You don't need `mode` fixture in all of tests in this file, remove it. It doubles the time to execute these tests.

Also, you can just reuse the existing test case for two examples here. `test_correct_ternary` is identical.

Reply:
So I removed the mode prefix, but I am unsure what you mean by test_correct_ternary is identical?